common-rest/src/test/java/uk/ac/ebi/uniprot/api/common/repository/search/SolrQueryRepositoryIT.java

```diff
@@ -50,7 +46,9 @@
     static void setUp() {
         try {
             SolrDataStoreManager solrStoreManager = new SolrDataStoreManager();
-            ClosableEmbeddedSolrClient solrClient = new ClosableEmbeddedSolrClient(SolrCollection.uniprot);
+            CoreContainer container = new CoreContainer(new File(System.getProperty(ClosableEmbeddedSolrClient.SOLR_HOME)).getAbsolutePath());
+            container.load();
+            ClosableEmbeddedSolrClient solrClient = new ClosableEmbeddedSolrClient(container, SolrCollection.uniprot);
```

Comment:
should we push lines 49 and 50 into `ClosableEmbeddedSolrClient`, so we don't need to create them here?